.github/BOTMETA.yml

```
@@ -689,7 +689,12 @@ files:
     maintainers: $team_google
     supershipit: $team_google
   $module_utils/gitlab.py: *gitlab
-  $module_utils/hwc_utils.py: *huawei
+  $module_utils/hwc_utils.py:
```

```suggestion
  $module_utils/hwc_utils.py: &huawei
```
If you add an anchor here . . .